packages/toolkit/utils/src/cli/patch.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
```

Comment:
This file should be placed in @modern-js/builder-shared.

Reply:
okk